[2025-03-31T12:51:20.791-0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-31T12:51:20.796-0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: exec_mult_tasks.1 manual__2025-03-31T17:51:17.031904+00:00 [queued]>
[2025-03-31T12:51:20.798-0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: exec_mult_tasks.1 manual__2025-03-31T17:51:17.031904+00:00 [queued]>
[2025-03-31T12:51:20.798-0500] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-31T12:51:20.935-0500] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): 1> on 2025-03-31 17:51:17.031904+00:00
[2025-03-31T12:51:20.937-0500] {standard_task_runner.py:72} INFO - Started process 404198 to run task
[2025-03-31T12:51:20.938-0500] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'exec_mult_tasks', '1', 'manual__2025-03-31T17:51:17.031904+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/exec_multiple_tasks.py', '--cfg-path', '/tmp/tmp12znrty1']
[2025-03-31T12:51:20.939-0500] {standard_task_runner.py:105} INFO - Job 27: Subtask 1
[2025-03-31T12:51:21.081-0500] {task_command.py:467} INFO - Running <TaskInstance: exec_mult_tasks.1 manual__2025-03-31T17:51:17.031904+00:00 [running]> on host royal-07.cs.wisc.edu
[2025-03-31T12:51:21.331-0500] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='cloud.user' AIRFLOW_CTX_DAG_ID='exec_mult_tasks' AIRFLOW_CTX_TASK_ID='1' AIRFLOW_CTX_EXECUTION_DATE='2025-03-31T17:51:17.031904+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-31T17:51:17.031904+00:00'
[2025-03-31T12:51:21.331-0500] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-31T12:51:21.338-0500] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-31T12:51:21.339-0500] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Task 1 executed"']
[2025-03-31T12:51:21.342-0500] {subprocess.py:99} INFO - Output:
[2025-03-31T12:51:21.342-0500] {subprocess.py:106} INFO - Task 1 executed
[2025-03-31T12:51:21.342-0500] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-31T12:51:21.452-0500] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-31T12:51:21.453-0500] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=exec_mult_tasks, task_id=1, run_id=manual__2025-03-31T17:51:17.031904+00:00, execution_date=20250331T175117, start_date=20250331T175120, end_date=20250331T175121
[2025-03-31T12:51:21.753-0500] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-31T12:51:21.763-0500] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-31T12:51:21.890-0500] {local_task_job_runner.py:245} INFO - ::endgroup::
